benefits/eligibility/templates/eligibility/includes/selection-label--veteran.html

```diff
@@ -7,8 +7,11 @@
 
 {% block description %}
   {% blocktranslate trimmed %}
-    This option is for people who have served in the active military, naval, or air service, and who were discharged or
-    released therefrom under conditions other than dishonorable. You will need to
-    <a href="https://www.va.gov/resources/verifying-your-identity-on-vagov/" target="_blank" rel="noopener noreferrer">verify your identity through VA.gov</a>.
+    This option is for people who have served in the active military, naval, or air service, and who were discharged or released therefrom under conditions other than dishonorable. You will need to verify your identity with
   {% endblocktranslate %}
+
+  {% include "core/includes/modal-trigger.html" with classes="border-0 bg-transparent p-0 login" modal="modal--login-gov-veterans" login=True period=True %}
+
+  {% include "eligibility/includes/modal--login-gov-help.html" with id="modal--login-gov-veterans" size="modal-lg" header="p-md-2 p-3" body="pb-md-3 mb-md-3 mx-md-3 py-0 pt-0 absolute-top" %}
```

Comment:
`veteran` singular in the modal id

Reply:
Nice catch! Done here: https://github.com/cal-itp/benefits/pull/1706/commits/57553fd19fa0bb7a6925dbc82e7d4d8f7bed444e

Ready for rereview